tests/Knp/Snappy/AbstractGeneratorTest.php

```diff
@@ -566,23 +566,25 @@ private function getPHPExecutableFromPath()
             return PHP_BINARY;
         }
 
-        $paths = explode(PATH_SEPARATOR, getenv('PATH'));
-        foreach ($paths as $path) {
-            // we need this for XAMPP (Windows)
-            if (strstr($path, 'php.exe') && isset($_SERVER['WINDIR']) && file_exists($path) && is_file($path)) {
-                return $path;
-            } else {
-                $php_executable = $path . DIRECTORY_SEPARATOR . 'php' . (isset($_SERVER['WINDIR']) ? '.exe' : '');
-                if (file_exists($php_executable) && is_file($php_executable)) {
-                    return $php_executable;
+        if (false !== getenv('PATH')) {
```

Comment:
I generally try to avoid nesting whole blocks of code in a `if` block (it can hinder readability and increase cyclomatic complexity) if it can be done in another way; here, I think you can use the early return technique:

```php
if (false !== getenv('PATH')) {
    return null;
}
```